week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -40,8 +40,26 @@
 function insertPeopleData(arrayOfPeople) {
   let content = document.querySelector("#content");
   //Write your code in here
+  //console.log(content);
+  
+  
+  for(var i = 0; i < arrayOfPeople.length; i++){
```

Comment:
```suggestion
  for(let i = 0; i < arrayOfPeople.length; i++){
```
Avoid use of the `var` keyword if possible - stick to `let` inside a `for` loop